### DISCUSSION TITLE
Welcome to tofu Discussions!

### DISCUSSION THREAD

**Discussion by Odomontois** [tofu-tf/tofu#475] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by susliko** [comment]
Hi! 🖖🏻 
Being zero-day tofu user, I'm very opimistic about the project and enjoy using it.
Would love to contribute!
